cmd/influx/main.go

```diff
@@ -53,8 +55,14 @@ func main() {
 	fs.StringVar(&c.Password, "password", c.Password, `password to connect to the server.  Leaving blank will prompt for password (--password="")`)
 	fs.StringVar(&c.Database, "database", c.Database, "database to connect to the server.")
 	fs.StringVar(&c.Format, "output", default_format, "format specifies the format of the server responses:  json, csv, or column")
+	fs.BoolVar(&c.Dump, "dump", false, "dump the contents of the given database to stdout")
 	fs.Parse(os.Args[1:])
 
+	if c.Dump {
+		c.connect("")
+		c.dump()
```

Comment:
We also might want to add this after prompt for password so if they want to do this interactively they could. Not 100% sure it matters that much, but it's an easy enough use case to support.